Code/GraphMol/Descriptors/AtomicEnvironmentVector.cpp

```diff
@@ -501,178 +501,178 @@ ArrayXXd AtomicEnvironmentVector(const ROMol &mol, int confId) {
+        radialAEV.row(i + 2), radialAEV.row(i + 3);
+    atomIdx++;
   }
 
-  ArrayXd even_closer_indices((distances.array() <= 3.5).count());
+  ArrayXd evenCloserIndices((distances.array() <= 3.5).count());
   int idx = 0;
-  for (int i = 0; i < distances.size(); i++) {
+  for (auto i = 0; i < distances.size(); i++) {
     if (distances(i) <= 3.5) {
-      even_closer_indices(idx) = i;
+      evenCloserIndices(idx) = i;
       idx++;
     }
   }
 
   // Angular Terms
 
-  ArrayXXd species12_angular(2, even_closer_indices.size());
-  ArrayXXd atom_index12_angular(2, even_closer_indices.size());
+  ArrayXXd species12Angular(2, evenCloserIndices.size());
+  ArrayXXd atomIndex12Angular(2, evenCloserIndices.size());
 
-  ArrayXXd vec_angular(even_closer_indices.size(), 3);
+  ArrayXXd vecAngular(evenCloserIndices.size(), 3);
 
-  species12_angular =
-      IndexSelect(species12_angular, species12, even_closer_indices, 1);
-  atom_index12_angular = IndexSelect(
-      atom_index12_angular, atom_index12_unflattened, even_closer_indices, 1);
-  vec_angular = IndexSelect(vec_angular, vec, even_closer_indices, 0);
+  species12Angular =
+      IndexSelect(species12Angular, species12, evenCloserIndices, 1);
+  atomIndex12Angular = IndexSelect(
+      atomIndex12Angular, atomIndex12Unflattened, evenCloserIndices, 1);
+  vecAngular = IndexSelect(vecAngular, vec, evenCloserIndices, 0);
 
-  auto n = even_closer_indices.size();
-  auto ret = TripleByMolecules(atom_index12_angular);
-  auto pair_index12 = ret.second;
-  auto central_atom_index = ret.first;
-  ArrayXXd sign12(2, pair_index12.cols());
+  auto n = evenCloserIndices.size();
+  auto ret = TripleByMolecules(atomIndex12Angular);
+  auto pairIndex12 = ret.second;
+  auto centralAtomIndex = ret.first;
+  ArrayXXd sign12(2, pairIndex12.cols());
 
-  for (int i = 0; i < pair_index12.rows(); i++) {
-    for (int j = 0; j < pair_index12.cols(); j++) {
-      if (pair_index12(i, j) < n) {
+  for (auto i = 0; i < pairIndex12.rows(); i++) {
+    for (int j = 0; j < pairIndex12.cols(); j++) {
+      if (pairIndex12(i, j) < n) {
         sign12(i, j) = 1;
       } else {
         sign12(i, j) = -1;
       }
     }
   }
 
-  n = atom_index12_angular.cols();
+  n = atomIndex12Angular.cols();
 
-  auto local_index = pair_index12.cast<int>();
-  pair_index12 = (local_index.array() - (local_index.array() / n).array() * n)
+  auto localIndex = pairIndex12.cast<int>();
+  pairIndex12 = (localIndex.array() - (localIndex.array() / n).array() * n)
                      .array()
                      .cast<double>();
 
-  ArrayXd pair_index12_flattened(2 * pair_index12.cols());
+  ArrayXd pairIndex12Flattened(2 * pairIndex12.cols());
   idx = 0;
-  for (int i = 0; i < pair_index12.rows(); i++) {
-    for (int j = 0; j < pair_index12.cols(); j++) {
-      pair_index12_flattened(idx) = pair_index12(i, j);
+  for (auto i = 0; i < pairIndex12.rows(); i++) {
+    for (int j = 0; j < pairIndex12.cols(); j++) {
```

Comment:
do you need to loop on all elements ie is it symetric ?

Reply:
```pairIndex12``` is not symmetric. The shape is ```2 * numPairs``` with each column containing a pair of atoms being considered.